src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+
+        for (int i = 0; i < records.length; i++) {
+            if (records[i].substring(0, records[i].indexOf(":")).equals(email)) {
```

Comment:
better to use a split method of the string and save result to String[] devidedRecord